### PR TITLE
Google Plus Removed

### PR DESCRIPTION
**Pull request by TheShubham99**
The contact page is added to the project.
It was missing before.

### CODE REVIEW

File: src/components/Contact/Contact.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.contact_bg .btn{
+  font-size: 18px;
+  color:rgba(40, 167, 69);
+}
+.mt-6e{
+  margin-top: 6rem;
+}
+section p{
+  font-size:15px;
+}
```
Comment (majordwarf) [inline src/components/Contact/Contact.css]: - There is no new line after CSS selectors, you did it for first few elements.
- Also no need to create new classes like `mt-6e`, we already have bootstrap module that can be used.
- Try keeping the class name simple and sensible, you are using double underscore in `.contact__bg-overlay`, you can write the following as `.contact__bg-overlay`.

File: src/components/Contact/Contact.jsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Row, Col } from 'reactstrap';
+import "./Contact.css";
+import { IconContext } from "react-icons";
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
```
Comment (majordwarf) [inline src/components/Contact/Contact.jsx]: We aren't using the Img module on the page, no need to import it.

File: src/components/Contact/Contact.jsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    },
+    community: file(relativePath: {eq: "images/8.jpg"}){
+      childImageSharp{
+        fluid(maxWidth: 300){
+          ...GatsbyImageSharpFluid_tracedSVG
+        }
+      }
+    }
+  }
+`);
```
Comment (majordwarf) [inline src/components/Contact/Contact.jsx]: We aren't using the queried images anywhere on the page, why query them?

File: src/components/Contact/Contact.jsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            <Col md="10" className="justify-content-center">
+              <h1>Contact Us</h1>
+            </Col>
+          </Row>
+        </div>
+      </div>
+      <section className="contact__details py-5 mt-6e">
+        
+      <Row>
+           <IconContext.Provider value={{ size: "6em"}}>
```
Comment (majordwarf) [inline src/components/Contact/Contact.jsx]: All of the icons are too large, you can have a look at other games' contact pages to get a gist.

**Review by majordwarf — COMMENTED** [review]
Left some comments. I guess you copied the code of About component thus there is a a lot of unused code like the graphQL queries, imports. I'd recommend installing ESLint plugin in your code/text editor, that'll help you regarding the above problems.

File: src/components/Contact/Contact.css (hunk 1 above)
Comment (TheShubham99) [reply src/components/Contact/Contact.css]: **Thank you very much @majordwarf for suggesting changes.
I have implemented the above-mentioned changes. :)**

File: src/components/Contact/Contact.jsx (hunk 3 above)
Comment (TheShubham99) [reply src/components/Contact/Contact.jsx]: Yes. Optimized :)

File: src/components/Contact/Contact.jsx (hunk 4 above)
Comment (TheShubham99) [reply src/components/Contact/Contact.jsx]: Resized 👍

**Comment by TheShubham99** [comment]
**Thanks @majordwarf 👍  :)
I have implemented all of the required changes. :)** 

Please review it again :)

**Review by TheShubham99 — COMMENTED** [review]
done

**Comment by TheShubham99** [comment]
**Thanks @majordwarf 👍  :)
I have implemented all of the required changes. :)** 

Please review it again :)

**Comment by TheShubham99** [comment]
@majordwarf I have just removed Google plus Links please review :)

**Review by TheShubham99 — COMMENTED** [review]
@majordwarf All changes are done. 
Google Plus Link removed. 👍

File: src/components/Contact/Contact.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+color: #606060;
+font-size: 12px;
+line-height: 1.5em;
+outline: 0;
+font-family: "Nunito Sans",sans-serif!important;
+box-sizing: border-box;
+zoom: 1;
+padding: 3rem 0;
+margin: 0 auto;
+text-align: center; }
```
Comment (majordwarf) [inline src/components/Contact/Contact.css]: Why are you overriding `.container` class here? This results breaking the theme of the whole website.
- Too much top and bottom padding.
- Things are centred which shouldn't be.
- Unnecessary CSS rulesets like `zoom`, `outline`.
- Also there is no new/empty line at the end of the file.
The above are just some of the initial observations.

image

File: src/components/Contact/Contact.jsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+              </a>
+              <a 
+                style={{padding:"5px"}}
+                className="link"
+                href="http://webchat.freenode.net/?channels=terasology&uio=d4?channels=%23terasology&nick=Terasologist...&prompt=1&useUserListIcons=true"
+              >
+                <center>  
+                  <FaComments style={{color:'#7bb32e'}} />
+                  <br />
+                  Webchat
```
Comment (majordwarf) [inline src/components/Contact/Contact.jsx]: It's IRC channel on freenode to be specific using Webchat's web UI. IRC would b a better name here.
Also, I don't know about the colourful icons, you should ask on Discord for community's input.

**Comment by TheShubham99** [comment]
@majordwarf Done. Removed Contact page & all other changes done.

**Comment by TheShubham99** [comment]
Will be doing the changes in fresh branch

**Pull request closed without merging**